Week-1/tanmay-sinha/README.md

```diff
@@ -0,0 +1,53 @@
+## About
+
+This is a basic python Tkinter program where user can type the phone name and then select phone from the suggestions(dropdown). It includes almost 1500 phones from which user can select the required phone. It appends phone details to the file "phone_details.json" every time user select a new phone.
+<br>
+<br>
+User will also receive desktop and mobile notifications and also E-mails regarding price drops and if price drops below the base price, at given time intervals. User can set the base price ( price at which user wants to purchase the phone ) and time interval at which user wants to check the price.
+
+Python version required - Python 3.6 and above.
+
+## Quick Setup
+
+1. Tkinter comes pre-installed with python. In case you have any problem using Tkinter , refer [this](https://stackoverflow.com/a/11690261).
+2. Install requirements by using
+```python 
+pip install -r requirements.txt
+```
+3. For setting up notify-run services ( desktop-mobile notifications ),
+refer this [Quick setup](https://notify.run/). User have to make his channel on the given link and the devices where he wants to receive notifications .
+
+4. Open "congif.json" file and update the email credentials . You have to allow less secure apps on your sender's Gmail account to send emails. Refer [this](https://hotter.io/docs/email-accounts/secure-app-gmail/).
+
+You are good to go.
+
+## How to Use?
+
+1. Run the python program GUIapp in the terminal. A window will appear like this.<br>
+  ![search box](https://github.com/tanmay-sinha/Git-Freeze/blob/tanmay-sinha/week1/Week-1/tanmay-sinha/images/Screenshot%20from%202019-12-19%2021-08-31.png)
```

Comment:
Not coding related but using relative paths for images would be better.

Since I cloned the repo and locally (w/o internet connectivity) I can't render .md right?